src/dune_engine/package.ml

```diff
@@ -675,6 +694,28 @@ let file ~dir ~name = Path.relative dir (Name.to_string name ^ opam_ext)
 let deprecated_meta_file t name =
   Path.Source.relative t.id.dir (Name.meta_fn name)
 
+let default name dir =
+  let depends =
+    let open Dependency in
+    [ { name = Name.of_string "ocaml"; constraint_ = None }
+    ; { name = Name.of_string "dune"; constraint_ = None }
+    ]
+  in
+  { id = { name = Name.make name; dir }
+  ; loc = Loc.none
+  ; version = None
+  ; synopsis = Some "A short synopsis"
+  ; description = Some "A longer description"
+  ; depends
+  ; conflicts = []
+  ; info = Info.empty
+  ; depopts = []
+  ; has_opam_file = false
+  ; tags = [ "topics"; "to describe"; "your"; "project" ]
```

Comment:
I include the `"to described"` bit to help show users how to do multi-word topics.

Reply:
That seems reasonable